scripts/prompts.ts

```diff
@@ -87,3 +93,115 @@ export async function promptPrepareRelease(opts: BuildOptions): Promise<PrepareR
   }
   return answers;
 }
+
+/**
+ * A type describing the answers to prompts for performing a release
+ */
+export type ReleasePromptAnswers = {
```

Comment:
Just a thought: Might be nice to break these types out into a separate file now that there are two type definitions and function definitions